debian/clearwater-etcd.init.d

```diff
@@ -261,12 +259,20 @@ verify_etcd_health()
           if [[ $rc != 0 ]]
           then
             /usr/bin/etcdctl member remove $local_member_id
-            if [[ $? == 0 ]]
-            then
-              rm -rf $DATA_DIR/$advertisement_ip
-            fi
+            rm -rf $DATA_DIR/$advertisement_ip
           fi
         fi
+
+        # If we only have one of local member id and data directory, remove it.
```

Comment:
Rather than say local member id and data directory, can you explain what this actually means and why it is bad? (That the etcd cluster thinks this node is a member already, or that this node has joined the etcd cluster in the past and won't go through the initial joining process again while it has stored data).